File: spec/code_samples/Perl_perl5/alerts@{list_type}/patch.pl
#!/usr/bin/perl
use Modern::Perl;
use Data::Dumper;
use HTTP::Request::JSON;
use LWP::UserAgent::JSON;
my $request = HTTP::Request::JSON->new("PATCH",
"'https://www.directfreight.com/api/alerts/%7Blist_type%7D?alert_id=SOME_STRING_VALUE'");
$request->content('{"days_of_week":{},"end_hour":{},"expires_after":{},"list_order":{},"list_type":{},"minute_frequency":{},"send_type":{},"start_hour":{},"time_zone":{}}');
my $browser = LWP::UserAgent::JSON->new();
my $response = $browser->request($request);
print Dumper($response->json_content);

File: spec/code_samples/Python_python3/alerts@{list_type}/patch.py
import http.client

conn = http.client.HTTPSConnection("www.directfreight.com")

payload = "{\"days_of_week\":{},\"end_hour\":{},\"expires_after\":{},\"list_order\":{},\"list_type\":{},\"minute_frequency\":{},\"send_type\":{},\"start_hour\":{},\"time_zone\":{}}"

conn.request("PATCH", "/api/alerts/%7Blist_type%7D?alert_id=SOME_STRING_VALUE", payload)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))